PureJava/org.eclipse.nebula.visualization.xygraph/src/org/eclipse/nebula/visualization/xygraph/figures/Axis.java

```diff
@@ -286,35 +313,29 @@ protected void paintClientArea(final Graphics graphics) {
 
 	/**
 	 * @return Range that reflects the minimum and maximum value of all traces
-	 *         on this axis. Returns <code>null</code> if there is no trace data
-	 *         or all trace ranges have NaNs or infinities.
+	 *         on this axis. Returns <code>null</code> if there is no trace
+	 *         data.
 	 */
 	public Range getTraceDataRange() {
 		double low = Double.POSITIVE_INFINITY;
 		double high = Double.NEGATIVE_INFINITY;
-		final boolean positiveOnly = isLogScaleEnabled();
```

Comment:
These changes will revert the fixes I put in [19bec72](https://github.com/DawnScience/swt-xy-graph/commit/19bec72c71e874eca7a8672c51c3f8947d036e2b) and [e155c2b](https://github.com/DawnScience/swt-xy-graph/commit/e155c2b10787960d8552d332a7211dc88cc567fc) that caused problems in plotting with log-scaled axes